syntax = "proto3";

message LeaveRoomS2C {
    int32 error = 1;
    bool succeed = 2;
    int32 uid = 3;
}